src/style.css

```diff
@@ -1 +1,119 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
```

Comment:
The same issue here. You've fixed the height of the container. It's better to let the content size dictate the container size unless there is a specific requirement.